neo4j/neo4j_test.go

```diff
@@ -0,0 +1,219 @@
+package neo4jstore
+
+import (
+	"context"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go/modules/neo4j"
+)
+
+var testStore *Storage
+
+// TestMain sets up and tears down the test container
+func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	// Start a Neo4j test container
+	neo4jContainer, err := neo4j.Run(ctx,
+		"neo4j:5.26",
```

Comment:
Move this to a `const` outside this function. That way it's easier to find/update in the future.